client/upload_client.go

```diff
@@ -0,0 +1,93 @@
+package client
+
+import (
+	"errors"
+	"io"
+	"os"
+
+	qs "github.com/yunify/qingstor-sdk-go/service"
+)
+
+type Uploader struct {
+	Ubucket  *qs.Bucket
```

Comment:
We will not expose the bucket, so just keep them private, ref: https://github.com/yunify/qingstor-sdk-go/blob/master/client/image/image.go#L99-L102